indico/modules/events/reminders/models/reminders.py

```diff
@@ -217,9 +218,10 @@ def send(self):
             attachments.append(MIMECalendar('event.ics', event_ical))
 
         for recipient in recipients:
-            email = make_email(
-                to_list=recipient, from_address=self.reply_to_address, template=email_tpl, attachments=attachments
-            )
+            params = dict(to_list=recipient, from_address=self.reply_to_address, template=email_tpl,
+                          attachments=attachments)
+            signals.event.reminder.before_reminder_make_email.send(params)
```

Comment:
I don't think the signal is very useful in the way it's currently used. There's not even a reference to the context where this happens. Some suggestions:

- Use `self` as the `sender`
- Pass other data that you just want to provide (ie immutable data) as individual kwargs
- I guess you want some of the data to be mutable? You could either use the "updates" pattern like in another recent PR from you guys, or we could see if there's a better option.

Especially because of the last point, I recommend including details with this type of pull request on how you intend to use a new signal.

Reply:
Hello @ThiefMaster, I have updated the PR with your recommended changes, for now, I'd just like to pass the `html=True` in the `make_email` function.
Thanks